lib/engines/content_block_manager/app/views/content_block_manager/content_block/shared/_cancel_delete_button.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_with url: url, method: :patch do %>
+  <%= render "govuk_publishing_components/components/button", {
+    text: "Cancel",
+    name: "_method",
```

Comment:
Could we have a hidden `_method` field within the form instead? Adding a name/value confused me for a sec

Reply:
hmm I spoke to soon cause now I'm getting issues with the controllers again! will take a look tomorrow

Reply:
so when I have a `delete` method on the form, or putting in a `hidden` input like below, I get a routing error on the feature test: `ActionController::RoutingError: No route matches [DELETE] {whatever path it is on at the time}` seems like it isn't taking on the `url` that I'm passing into it, or maybe having two forms on the page is conflicting 🤔 
If it isn't a major issue at the moment I will pop the issue in the commit message for now.
`<input type="hidden" name="_method" value="delete" autocomplete="off">`

Reply:
Ah, I wonder if the issue is because we have two nested forms? I seem to remember that being an issue in the past

Reply:
I wonder whether you could get rid of the nested form and go back to having the `name` and `_method` on the button - after all, sending a `DELETE` in browser is just a `POST` request that's faked by sending a `_method=DELETE` param